lib/validation/index.js

```diff
@@ -23,15 +23,14 @@ function applyValidator(validator, key, value) {
         }
     }
 
-    let args = [];
-    if (_.isArray(validator.arguments)) {
-        args = validator.arguments;
-    } else if (validator.arguments !== undefined) {
-        args = [ validator.arguments ];
+    if (validator.arguments === undefined) {
+        validator.arguments = [];
+    } else if (!_.isArray(validator.arguments)) {
```

Comment:
Would it be useful to have an additional `else if` block with a `_.isArguments` here?

Reply:
No, because these are not actual function arguments here.